Client/components/PasswordResetConfirm.jsx

```diff
@@ -0,0 +1,118 @@
+import React, {Component} from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom';
+import '../public/style.css';
+import TitleBar from './TitleBar';
+import $ from '../public/jquery.js';
+import AppActions from '../actions/AppActions';
+import AppStore from '../stores/AppStore';
+import QueryString from 'query-string';
+import {green100, green500, green700} from 'material-ui/styles/colors';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+const style = {margin: 5};
+const muiTheme = getMuiTheme({
+  palette: {
+    primary1Color: green500,
+    primary2Color: green700,
+    primary3Color: green100,
+  },
+}, {
+  avatar: {
+    borderColor: null,
+  },
+});
+
+/**
+ * Redirects to home page after resetting password
+ * @return {void} returns void
+ */
+function redirectToHome() {
+  setTimeout(() => {
+       window.location.href = "/#"; //will redirect to your blog page (an ex: blog.html)
+    }, 2000); //will call the function after 2 secs.
```

Comment:
Using a timeout is not advisable. Whatever you were trying to achieve, using a callback may be better.